maas/resource_maas_vm_host.go

```diff
@@ -110,13 +117,20 @@ func resourceMAASVMHost() *schema.Resource {
 					},
 				},
 			},
+			"key": {
+				Type:          schema.TypeString,
+				Optional:      true,
```

Comment:
Should we consider adding `Computed: true` as wel to cover the case when we only provide the trust password and MAAS is creating a cert/key combo for the VM host?

Reply:
of course, in that case we need to also update the read function accordingly.